plugins/module_utils/network/ios/config/snmp_server/snmp_server.py

```diff
@@ -34,6 +34,20 @@
 )
 
 
+def handle_deprecates(wantd):
```

Comment:
Amazing!. Could you please remove this function from here

Reply:
Oups.... 😄 
Removed